src/App.js

```diff
@@ -0,0 +1,20 @@
+import React, { Component } from 'react';
+import './App.css';
+
+class App extends Component {
+    render() {
+        return (
+            <div className="App">
+                <div className="App-header">
+                    <h2>Car Maitenance Tracker</h2>
+                </div>
+                <br/>
```

Comment:
As a general rule, never use `<br />`.  `br` is purely presentational and can be handled by the CSS.  Mixing presentation with your markup breaks separation of concerns.